test/e2e/ps_test.go

```diff
@@ -476,5 +476,13 @@ var _ = Describe("Podman ps", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.OutputToString()).To(ContainSubstring("echo very long cr..."))
 	})
+	It("podman ps --format {{RunningFor}}", func() {
+		_, ec, _ := podmanTest.RunLsContainer("")
+		Expect(ec).To(Equal(0))
 
+		result := podmanTest.Podman([]string{"ps", "-a", "--format", "{{.RunningFor}}"})
+		result.WaitWithDefaultTimeout()
+		Expect(result.ExitCode()).To(Equal(0))
+		Expect(result.OutputToString()).To(ContainSubstring("ago"))
```

Comment:
Is there an `EndsWith()` function in ginkgo? If so, for a future PR (absolutely not for this one, it's not worth the CI respin cost), might I suggest `EndsWith(" ago")`? Note the space, too. The current test would match on "paragon" and "agony" - not that those are likely to appear in this context, but hey.